### ISSUE TITLE
[Bug]: Blank screen shortly after use - Intel Mac

### ISSUE THREAD

**Issue by rylan-michael** [httpie/desktop#12]
### What happened?

I started using the HTTPie Desktop application for Intel Macs and after about 10 minutes after the first run the application went blank. I haven't been able to fix the issue and have reinstalled. Is there a way for me to clear caches or reset state?

![image](https://user-images.githubusercontent.com/13746934/163824861-0e0ca5ca-dc15-4a70-bb48-05bb96b2bd7d.png)

### Steps to reproduce?

```Markdown
Not sure it can be reproduced but...

1. Installed application.
2. Opened and began exploring the features like cycling through request previews and collections.
3. After about 10 min screen goes blank.
```

### What did you expect to happen?

I expected the application to return a user interface after force refreshing, or closing and reopening the application.

### Platform

macOS

### HTTPie Desktop version

2022.6.0

### Relevant log output

```Shell
dependencies.js:15 TypeError: Cannot read properties of undefined (reading 'toUpperCase')
    at dependencies.js:15
    at Array.map (<anonymous>)
    at Function.formatHeaderName (dependencies.js:15)
    at Function.groupHeadersForDisplay (dependencies.js:15)
    at Function.getHeaderLines (dependencies.js:15)
    at Function.getHeaders (dependencies.js:15)
    at q.getHeaders (dependencies.js:15)
    at q.getHeadersWithStartLine (dependencies.js:15)
    at q.toString (dependencies.js:15)
    at Cj (dependencies.js:15)
    at ia (dependencies.js:15)
    at qs (dependencies.js:15)
    at kc (dependencies.js:15)
    at Sc (dependencies.js:15)
    at Ec (dependencies.js:15)
    at mc (dependencies.js:15)
    at dependencies.js:15
    at t.unstable_runWithPriority (dependencies.js:15)
    at Ho (dependencies.js:15)
    at Go (dependencies.js:15)
    at Ko (dependencies.js:15)
    at lc (dependencies.js:15)
    at Ta (dependencies.js:15)
    at dependencies.js:15
    at dependencies.js:15
    at Object.next (dependencies.js:15)
    at a (dependencies.js:15)
(anonymous) @ dependencies.js:15
index.js:2 Refused to load the script 'https://cdn.s.httpie.io/next-integrations/middleware/analytics.js-middleware-braze-deduplicate/latest/analytics.js-middleware-braze-deduplicate.js.gz' because it violates the following Content Security Policy directive: "script-src 'self' data: 'unsafe-eval' 'unsafe-inline'". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jakubroztocil** [comment]
Thank you for the report, @rylan-michael! We’re looking into it 👀

**Comment by jakubroztocil** [comment]
We’ve found the root cause (header names with a trailing `-`) and fixed the issue. A new release is coming later today or tomorrow latest. 

In the meantime, here’s how to fix it locally:

<img width="932" alt="image" src="https://user-images.githubusercontent.com/326885/164206367-5b13dffd-2b99-46f5-8f82-8bffb3e720ff.png">


1. On the desktop, open DevTools (`⌘` + `⌥` + `I`)
2. Go to the `Application` tab
3. Select `Local storage` / `file://`
4. Type `-"` in the filter field
5. In the filtered row’s value field, replace `-"` with `"` (you can do that in your editor)
6. Save the value
7. Reload the app (`⌘` + `R`)

**Comment by rylan-michael** [comment]
That worked and I'm back in action 🎉

**Comment by jakubroztocil** [comment]
@rylan-michael a quick update: We’ve [just released 2022.7.0](https://httpie.io/blog/changelog-0014) so this issue should be officially gone now.
